### PR TITLE
Fix API Changes to Linode Services

### PR DESCRIPTION
**Pull request by na3d**

### CODE REVIEW

File: src/linodes/components/Linode.js [hunk 1]
```diff
@@ -28,10 +28,14 @@ renderPowerButton.propTypes = {
   onReboot: PropTypes.func,
 };
 
-export function renderPlanStyle(planStr) {
-  const plan = planStr.split(' ');
-
-  return `${plan[0]} ${plan[1] / 1024}G`;
+export function renderPlanStyle(services) {
+  for (const serv of services) {
+    if (serv.service_type && serv.service_type === 'linode') {
```
Comment (SirCmpwn) [inline src/linodes/components/Linode.js]: I don't think this works the way you think it does. `serv` is a key from the services dict, not the value.

Comment (na3d) [inline src/linodes/components/Linode.js]: Services array now, swagger was having issues...

Comment (na3d) [inline src/linodes/components/Linode.js]: This is honestly just a patch job, because services are about to change again soon.

**Comment by na3d** [comment]
Needs a test for when a Linode doesn't have a Linode service.

File: src/linodes/components/Linode.js (hunk 1 above)
Comment (SirCmpwn) [inline src/linodes/components/Linode.js]: I just did a GET /services/linodes and it still looks like a standard list response. Which means that api-store would store it as a dict keyed by IDs in the state, right? `for (const foo of bar)` in javascript iterates over the keys of `bar`, so if `bar` was `{ a: 1, b: 2, c: 3 }`, `foo` would be 'a', 'b', and 'c'.

File: src/linodes/components/Linode.js [hunk 2]
```diff
@@ -28,10 +28,14 @@ renderPowerButton.propTypes = {
...
+      const plan = serv.label.split(' ');
+      return `${plan[0]} ${plan[1] / 1024}G`;
+    }
+  }
+  return false;
```
Comment (SirCmpwn) [inline src/linodes/components/Linode.js]: Misunderstood what this is for. Let me propose an alternate implementation:

    return _.filter(services, s => s.service_type === 'linode')
      .reduce((a, s) => {
        const plan = serv.label.split(' ');
        return a + `${plan[0]} ${plan[1] / 1024}G`;
      }, '');

**Pull request merged**